packages/files/src/index.ts

```diff
@@ -0,0 +1,57 @@
+import { Restroom } from "@restroom-mw/core";
+import { ObjectLiteral } from "@restroom-mw/core/src/types";
+import { Zencode } from "@restroom-mw/zencode";
+import axios from "axios";
+import { NextFunction, Request, Response } from "express";
+import fs from 'fs'
+import https from "https";
+import {
+  DOWNLOAD,
+} from "./actions";
+import yauzl from 'yauzl';
+import extract from 'extract-zip';
+
+const TEMP_FILE = '/tmp/temp-zip-file-downloaded-from-restroom';
+
+export default (req: Request, res: Response, next: NextFunction) => {
+  const rr = new Restroom(req, res);
+  let content: ObjectLiteral = {};
+
+  rr.onBefore(
+    async (params: {
+      zencode: Zencode;
+      keys: ObjectLiteral;
+      data: ObjectLiteral;
+    }) => {
+    }
+  );
```

Comment:
We can remove this... is useless right?

Reply:
Yes it should be, this mw should operate only in on.success